packages/next/src/server/config-shared.ts

```diff
@@ -612,6 +612,17 @@ export interface ExperimentalConfig {
    * Enables the use of the `"use cache"` directive.
    */
   useCache?: boolean
+
+  /**
+   * When enabled, will detect and report slow modules in the dev build in the terminal.
+   */
+  slowModuleDetection?: {
+    /**
+     * Time threshold in milliseconds for considering a module "slow".
+     * Modules taking longer than this to build will be reported.
+     */
+    slowModuleThresholdMs: number
```

Comment:
feel we missed "what is the module doing" in the threshold config. Maybe `loadingThresholdMs` indicates it's about module loading time of the modules. Also avoid duplication of the naming when we add nested configs.